Foundation/FileHandle.swift

```diff
@@ -342,6 +342,14 @@ open class FileHandle : NSObject, NSSecureCoding {
         }
 #endif
     }
+    
+    internal func _readBytes(into buffer: UnsafeMutablePointer<UInt8>, length: Int) throws -> Int {
+        let amtRead = _read(_fd, buffer, length)
```

Comment:
Hmm, is `_read` the internal method?  Shouldn't that just throw if `read` returned `< 0`?  `_read` is also a UCRT function on Windows, so please use `self._read` to make it more obvious.

Reply:
This `_read` is not seems to be an internal method, it is a reference of either Darwin.read(_:_:_:) or Glibc.read(_:_:_:) method.

Reply:
Since I am do not have experience in Windows, I just want to run by you the following implementation for `_readBytes()` method,

```swift
    internal func _readBytes(into buffer: UnsafeMutablePointer<UInt8>, length: Int) throws -> Int {
#if os(Windows)
        var BytesRead: DWORD = 0
        let BytesToRead: DWORD = DWORD(length)
        if ReadFile(_handle, buffer, BytesToRead, &BytesRead, nil) == FALSE {
            throw _NSErrorWithWindowsError(GetLastError(), reading: true)
        }
        return Int(BytesRead)
#else
        let amtRead = _read(_fd, buffer, length)
        if amtRead < 0 {
            throw _NSErrorWithErrno(errno, reading: true)
        }
        return amtRead
#endif
    }
```

Reply:
If the above code implementation works, I will update the method as well.

Reply:
Yeah, the use of `ReadFile` is fine.

Reply:
Yeah, that implementation is fine.